README.md

```diff
@@ -54,9 +54,19 @@ npm install -g oav
 ```
 bash-3.2$ oav -h
 Commands:
-  validate-example <spec-path>  Performs validation of x-ms-examples and
-                                examples present in the spec.
-  validate-spec <spec-path>     Performs semantic validation of the spec.
+  extract-xmsexamples <spec-path>           Extracts the x-ms-examples for a
```

Comment:
Currently by default it does sorting alphabetically. Quickly looking at [commandDir](https://github.com/yargs/yargs#commanddirdirectory-opts) I do not see the straight forward option yet. May be i'll just leave it for now.